README.md

```diff
@@ -31,17 +31,52 @@ __gtk__ expects __GTK+__, __GLib__ and __Cairo__ development files to be install
 
 ### Windows
 
+Make sure you have the [GNU
+ABI](https://www.rust-lang.org/downloads.html#win-foot) version of the rust
+compiler installed. Since we need our own toolchain make sure to uncheck
+"Linker and platform libraries" in the Rust setup.
+![Screenshot](rust_setup.png)
+
+If you already have installed Rust, check if your Rust installation has
+`gcc.exe` and `ld.exe` in its `bin` directory. In that case remove those
+executables, they will be provided by mingw instead. If you won't you'll
+probably get a linking error `ld: cannot find -limm32`.
+
+For the toolchain one has two possibilities:
+ 1. Install msys2 with a mingw toolchain and the gtk3 package
+ 2. Install the mingw-w64 toolchain separately and then install a GTK+ SDK
+
+#### 1. msys2 toolchain
+
+This method is recommended according to the [GTK+ project]
+(http://www.gtk.org/download/windows.php). You can follow [this guide]
+(https://blogs.gnome.org/nacho/2014/08/01/how-to-build-your-gtk-application-on-windows/)
+or follow these steps:
+
+ - Install and update [msys2](https://msys2.github.io/)
+ - Install the mingw toolchain:
+
+ ```Shell
+ $ pacman -S mingw-w64-x86_64-toolchain
```

Comment:
Does this actually work? I've tried this in appveyor and it didn't because the msys2 mingw is incorrect flavor, it's not win32+dwarf/seh. I ended up [using mingw-builds](https://github.com/gkoz/gtk-rs-gtk/commit/faab1070353fbb01f855eecc833bb8592957672e), the installer linked at http://mingw-w64.org/doku.php/download/mingw-builds